doc/selectors.rst

```diff
@@ -28,26 +28,28 @@ string patterns in, CadQuery will automatically use the associated selector obje
 Combining Selectors
 ==========================
 
-Selectors can be combined arithmetically and logically, so that it is possible to do intersection, union, and other
-combinations.  For example::
+Selectors can be combined logically, currently defined operators include **and**, **or**, **not** and **exc[ept]** (set difference).  For example:
 
-    box = cadquery.Workplane("XY").box(10,10,10)
+.. cq_plot:: 
```

Comment:
cq_plot is probably something that @dcowden added during the documentation refactor, but I'm not sure what it is. It shows up in the rendered documentation for me as cq_plot still.